kolibri/core/content/utils/annotation.py

```diff
@@ -134,6 +135,93 @@ def fix_multiple_trees_with_id_one():
             )
 
 
+def update_num_coach_contents():
+    """
+    Function to set num_coach_content on topic trees that were imported before
+    the annotation would have been run.
+    """
+    bridge = Bridge(app_name=CONTENT_APP_NAME)
+
+    ContentNodeClass = bridge.get_class(ContentNode)
+
+    ContentNodeTable = bridge.get_table(ContentNode)
+
+    connection = bridge.get_connection()
+
+    child = ContentNodeTable.alias()
```

Comment:
what kind of object is this, and what's it a child of? I see 

```python
    def get_table(self, DjangoModel):
        return self.get_class(DjangoModel).__table__
```

but I can't find documentation for `Model.__table__` or `Model.__table__.alias`

Reply:
It is a SQLAlchemy table object. https://docs.sqlalchemy.org/en/13/orm/extensions/declarative/table_config.html#using-a-hybrid-approach-with-table

Reply:
I never dared to dream that someone else might want to touch this code, I will dream!